clang/lib/DPCT/ASTTraversal.cpp

```diff
@@ -13529,38 +13529,44 @@ void NamespaceRule::runRule(const MatchFinder::MatchResult &Result) {
     Toklen = Lexer::MeasureTokenLength(
         End, SM, DpctGlobalInfo::getContext().getLangOpts());
     Len = SM.getFileOffset(End) - SM.getFileOffset(Beg) + Toklen;
-    auto Iter = MapNames::MathFuncNameMap.find(UD->getNameAsString());
-    if (Iter != MapNames::MathFuncNameMap.end()) {
-      DpctGlobalInfo::getInstance().insertHeader(UD->getBeginLoc(), HT_Math);
-      std::string Repl{"using "};
-      Repl += Iter->second;
+
+    std::unordered_set<const clang::UsingShadowDecl *> IntroducedUSD;
+    for (const auto &child : UD->getDeclContext()->decls()) {
```

Comment:
Please try to merge the two for(){} to make code simpler.

Reply:
Done